test/functional/apps/discover/group1/_discover_histogram.ts

```diff
@@ -66,13 +66,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     it('should modify the time range when the histogram is brushed', async function () {
       await PageObjects.common.navigateToApp('discover');
       await PageObjects.discover.waitUntilSearchingHasFinished();
-      // this is the number of renderings of the histogram needed when new data is fetched
-      let renderingCountInc = 1;
```

Comment:
ℹ️ This is not needed anymore, it seems that the situation improved here, possibly by simplifying the unified search code

Reply:
Nice! Started a quick flaky test check here https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/6660